backends/dnf/pk-backend-dnf.c

```diff
@@ -150,12 +151,32 @@ pk_backend_setup_dnf_context (DnfContext *context, GKeyFile *conf, const gchar *
 	dnf_context_set_cache_dir (context, cache_dir);
 	solv_dir = g_build_filename (destdir, "/var/cache/PackageKit", release_ver, "hawkey", NULL);
 	dnf_context_set_solv_dir (context, solv_dir);
-	repo_dir = g_build_filename (destdir, "/etc/yum.repos.d", NULL);
-	dnf_context_set_repo_dir (context, repo_dir);
```

Comment:
can we do this as one commit, and the destdir stuff in another?

Reply:
it's an atomic change as we're switching from one directory to multiple directories. destdir is already there...

Reply:
Ahh okay. If it's warning free I guess it LGTM.